awx/main/utils/handlers.py

```diff
@@ -176,7 +176,7 @@ def __init__(self, fqdn=False, message_type=None, username=None, password=None,
         self.verify_cert = verify_cert
         super(BaseHTTPSHandler, self).__init__(**kwargs)
         self.session = FuturesSession(executor=VerboseThreadPoolExecutor(
-            max_workers=2  # this is the default used by requests_futures
+            max_workers=1  # changed value from the default to fix python threading bug
```

Comment:
I'm being super nitpicky now over a one-line change, but for future generations, do you mind linking to the actual bug (https://bugs.python.org/issue32576)

😄

Reply:
cool added that, apologies I thought about adding it but didn't want to make a super huge comment